libs/angular-generator/src/lib/fundamental-styles/fundamental-styles-component-file.ts

```diff
@@ -0,0 +1,68 @@
+import {AngularGeneratedFile} from "../angular-generated-file";
+import {AngularGeneratorOptions} from "../angular-generator-options";
+import {format} from "prettier";
+import {parse, join, dirname, basename} from "path";
+import {pascalCase} from "./helpers";
+import {AngularExportSpecifierType} from "../angular-export-specifier-type";
+
+export class FundamentalStylesComponentFile extends AngularGeneratedFile {
```

Comment:
Inconsistent: The class itself generates directive. But the name of the generator class contains `Component`